autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectColumn.tsx

```diff
@@ -180,7 +179,9 @@ const CourseSelectColumn: React.FC = () => {
     if (card) {
       // Grow this card if it is focused so that
       // it can be viewed properly on low resolutions
-      const isExpandedRow = (card.collapsed === false);
+      const isExpandedRow = (card.collapsed === false
+        && !card.loading
+        && card.course);
```

Comment:
This is because we want a different message depending on if it has no sections vs. if there is no course selected. Previously this was done both here and in SectionSelect, but I moved both here since its easier.

Reply:
This is because we want a different message depending on if it has no sections vs. if there is no course selected. Previously this was done both here and in SectionSelect, but I moved both here since its easier.